common/test/acceptance/tests/lms/test_problem_types.py

```diff
@@ -0,0 +1,752 @@
+"""
+Bok choy acceptance and a11y tests for problem types in the LMS
+
+See also lettuce tests in lms/djangoapps/courseware/features/problems.feature
+"""
+import random
+import textwrap
+
+from abc import ABCMeta, abstractmethod
+from nose.plugins.attrib import attr
+from selenium.webdriver import ActionChains
+
+from capa.tests.response_xml_factory import (
```

Comment:
Actually I do want to push back a bit on your argument about it 'just being in a different path', because that's not what I was after. To me it's a delineation between test responsibilities and keeping your test code properly decoupled from your production code. It can be a bad pattern to cannibalize production code to make it easier to write tests. I've certainly seen instances where tests are not catching an issue because they are assuming certain parts of production code in order to stay as DRY as possible. 

In this case, we are talking about the difference between unit and acceptance tests, which is different than test vs prod-code. So I was curious if that barrier had been broken yet. (I think it has in this repo, even with the XBlockFixtureDesc if I remember correctly.)

Just wanted to be clear that I was properly motivated in my line of questioning here.